apps/01PigLatin.js

```diff
@@ -7,8 +7,43 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
```

Comment:
I'll be able to fix that tomorrow. Did not understand assignment until this
evening.

On Jul 31, 2016 7:58 PM, "maddiesolz" <notifications@github.com> wrote:

> In apps/01PigLatin.js
> <https://github.com/AustinCodingAcademy/intermediate-workbook/pull/60#discussion_r72913793>
> :
>
> > @@ -7,8 +7,43 @@ prompt.start();
> >
> >  function pigLatin(word) {
> >
> > -    // Your code here
>
> Hi Rebecca! I think you also have to answer some of the concept questions.
> They can be found here:
>
> https://gist.github.com/paulrenenichols/099c38009a569d603a098060f6cc3018
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AustinCodingAcademy/intermediate-workbook/pull/60/files/79988d7f609877276274b0c7926ea6399972f621..b49ce3f3746f85c8bb4aa9270fc96cb01a69a774#r72913793>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOXlVhuBO62MUXtIb5mHqQuam4IUAWtlks5qbTaTgaJpZM4JUtcP>
> .
>